util/doxygen/Doxyfile

```diff
@@ -0,0 +1,403 @@
+# Doxyfile 1.8.17
```

Comment:
This is `doxygen -g` with some overrides, right? Can we just include the values that have changed from the defaults?

Reply:
I'll see what I can do :)

Reply:
Done